lib/auth/register.go

```diff
@@ -91,6 +92,10 @@ type RegisterParams struct {
 	CAPath string
 	// GetHostCredentials is a client that can fetch host credentials.
 	GetHostCredentials HostCredentials
+	// Time specifies the time as number of seconds since epoch to use in
+	// client for verifying TLS certificates.
```

Comment:
```suggestion
	// Time specifies the current time to use in client for verifying TLS certificates.
```

Reply:
nit: I actually copied the [comment](https://github.com/golang/go/blob/9b955d2d3fcff6a5bc8bce7bafdc4c634a28e95b/src/crypto/tls/common.go#L491-L493) from the `tls.Config`